serfclient/connection.py

```diff
@@ -134,3 +134,28 @@ def _counter(self):
     def _error_message(self, exception):
         return "Error %s connecting %s:%s. %s." % \
             (exception.args[0], self.host, self.port, exception.args[1])
+
+    def _decode_addr_key(self, obj_dict):
+        """
+        Callback function to handle the decoding of the 'Addr' field.
+
+        Serf msgpack 'Addr' as an IPv6 address, and the data needs to be unpack
+        using socket.inet_ntop().
+
+        See: https://github.com/KushalP/serfclient-py/issues/20
+
+        :param obj_dict: A dictionary containing the msgpack map.
+        :return: A dictionary with the correct 'Addr' format.
+        """
+        key = b'Addr'
+        if key in obj_dict:
+            ip_addr = socket.inet_ntop(socket.AF_INET6, obj_dict[key])
```

Comment:
Thanks @TechBK. You are right, I consulted the Python doc for [socket.inet_ntop()](https://docs.python.org/2/library/socket.html#socket.inet_ntop) and ran the following test. 

In summary, a ValueError could be raised. I will submit a PR to fix.
```
>>> import socket
>>> socket.inet_ntop(socket.AF_INET6, '192.168.0.1')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: invalid length of packed IP address string
```